004-quarkus-HHH-and-HV/README.md

```diff
@@ -1,3 +1,7 @@
 # Quarkus - Hibernate and Hibernate Validator (lazy fetch strategy)
 This scenario uses in-memory Java database H2.  
-Test class annotated with `@QuarkusTestResource(H2DatabaseTestResource.class)` starts an in-memory H2 database
\ No newline at end of file
+Test class annotated with `@QuarkusTestResource(H2DatabaseTestResource.class)` starts an in-memory H2 database
+
+Module that covers intergation with some Hibernate features like:
+- Usage of Code Generator: `hibernate-jpamodelgen` dependency
```

Comment:
Adjust README pls. 
I'm thinking it will be great to point out that `items` and `validator` packages are serving different use cases and not interconnected in any way

Reply:
Removed this line. Thanks!